pkg/api/datasources.go

```diff
@@ -191,22 +192,23 @@ func GetDataSourceIdByName(c *middleware.Context) Response {
 
 func convertModelToDtos(ds *m.DataSource) dtos.DataSource {
 	dto := dtos.DataSource{
-		Id:                ds.Id,
-		OrgId:             ds.OrgId,
-		Name:              ds.Name,
-		Url:               ds.Url,
-		Type:              ds.Type,
-		Access:            ds.Access,
-		Password:          ds.Password,
-		Database:          ds.Database,
-		User:              ds.User,
-		BasicAuth:         ds.BasicAuth,
-		BasicAuthUser:     ds.BasicAuthUser,
-		BasicAuthPassword: ds.BasicAuthPassword,
-		WithCredentials:   ds.WithCredentials,
-		IsDefault:         ds.IsDefault,
-		JsonData:          ds.JsonData,
-		SecureJsonFields:  map[string]bool{},
+		Id:                     ds.Id,
+		OrgId:                  ds.OrgId,
+		Name:                   ds.Name,
+		Url:                    ds.Url,
+		Type:                   ds.Type,
+		Access:                 ds.Access,
+		Password:               ds.Password,
+		Database:               ds.Database,
+		User:                   ds.User,
+		BasicAuth:              ds.BasicAuth,
+		BasicAuthUser:          ds.BasicAuthUser,
+		BasicAuthPassword:      ds.BasicAuthPassword,
+		WithCredentials:        ds.WithCredentials,
+		IsDefault:              ds.IsDefault,
+		JsonData:               ds.JsonData,
+		SecureJsonFields:       map[string]bool{},
+		MinAutoRefreshInterval: ds.MinAutoRefreshInterval,
```

Comment:
do we really need a new column for this? Maybe we can use a property on the jsonData